src/domain/operation_mode.py

```diff
@@ -80,3 +85,20 @@ def stop_ftp_server(self):
             FTPsServer.ftps_server.server.close_all()
             FTPsServer.ftps_server.server.close()
             self.ftp_thread.join()
+
+    def _scheduled_update_actuators_trigger(self):
+        while not self.flag_stop_update_actuators_thread:
+            time.sleep(10)
+            self.update_actuator_status.emit()
+
+    def start_update_actuators_thread(self):
+        update_actuators_thread = threading.Thread(target=self._scheduled_update_actuators_trigger)
+        if update_actuators_thread:
+            update_actuators_thread.start()
+            logger.info("Starting thread for update actuators view...")
+        else:
+            logger.error("Unable to create new thread for update actuators view.")
+        return update_actuators_thread
+
+    def stop_update_actuators_thread(self):
+        self.flag_stop_update_actuators_thread = True
```

Comment:
move here logic to stop the actuators thread directly. we could get rid of the flag.